.eslintrc.cjs

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    sourceType: 'module',
+    project: [
+      './packages/*/tsconfig.json',
+      './packages/*/jsconfig.json',
+      './packages/wallet/*/jsconfig.json',
+    ],
+    tsconfigRootDir: __dirname,
+    extraFileExtensions: ['.cjs'],
+  },
+  plugins: [
+    '@jessie.js/eslint-plugin',
+    '@typescript-eslint',
+    'eslint-plugin-import',
+    'eslint-plugin-prettier',
+  ],
+  rules: {
+    '@typescript-eslint/prefer-ts-expect-error': 'warn',
+    'jsdoc/no-multi-asterisks': 'off',
+    'jsdoc/multiline-blocks': 'off',
+    // Use these rules to warn about JSDoc type problems, such as after
+    // upgrading eslint-plugin-jsdoc.
+    // Bump the 1's to 2's to get errors.
+    // "jsdoc/valid-types": 1,
+    // "jsdoc/no-undefined-types": [1, {"definedTypes": ["never", "unknown"]}],
+    'jsdoc/tag-lines': 'off',
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: [
+          '**/*.config.js',
+          '**/*.config.*.js',
+          '**/*test*/**/*.js',
+          '**/demo*/**/*.js',
+          '**/scripts/**/*.js',
+        ],
+      },
+    ],
+    'prettier/prettier': 'warn', // @jessie.js/recommended errors. This repo checks format separately but keeps warn to maintain that "eslint --fix" prettifies
```

Comment:
Can you move the `@jessie.js` comment to a separate line?

Reply:
Hoping to get this in which will obviate the "jessie" part https://github.com/Agoric/eslint-config-jessie/pull/28

There's still the "--fix" issue which I'll punt until https://github.com/Agoric/agoric-sdk/issues/4339